src/Sylius/Component/Core/spec/Promotion/Filter/ProductFilterSpec.php

```diff
@@ -53,4 +53,9 @@ function it_returns_all_items_if_configuration_is_invalid(OrderItemInterface $it
     {
         $this->filter([$item], [])->shouldReturn([$item]);
     }
+
+    function it_returns_all_items_if_configuration_is_empty(OrderItemInterface $item)
+    {
+        $this->filter([$item], ['filters' => ['products_filter' => []]])->shouldReturn([$item]);
```

Comment:
More apprioprate would be `['filters' => ['products_filter' => ['products' => []]]]`